### PR TITLE
Upgrade to latest EventHub nuget package so the EventHub output work with .Net 5

### PR DESCRIPTION
**Pull request by yantang-msft**
This is to address this issue: https://github.com/Azure/diagnostics-eventflow/issues/377

### CODE REVIEW

File: src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <Description>Provides an output implementation that sends diagnostics data to Azure Event Hubs.</Description>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
     <Authors>Microsoft</Authors>
-    <TargetFrameworks>net452;netstandard1.6;netstandard2.0</TargetFrameworks>
```
Comment (karolz-ms) [inline src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj:7]: We could also keep producing net452 binary (upgrade the dependency to ver 1.0.3), and refer to version 4.3.1 for netstandard 1.6 and 2.0. .NET 4.5.2 is still supported so maybe it is too early to discontinue that...

File: src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
+    <TargetFrameworks>netstandard1.6;netstandard2.0</TargetFrameworks>
     <AssemblyName>Microsoft.Diagnostics.EventFlow.Outputs.EventHub</AssemblyName>
     <VersionPrefix>1.6.0</VersionPrefix>
```
Comment (karolz-ms) [inline src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj:9]: You want to increment the package version to something like 1.7.0

File: test/Microsoft.Diagnostics.EventFlow.Outputs.Tests/Microsoft.Diagnostics.EventFlow.Outputs.Tests.csproj [hunk 3]
```diff
@@ -20,15 +20,15 @@
...
+  <ItemGroup Condition=" '$(TargetFramework)' == 'net471' ">
     <Reference Include="System" />
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
 
   <ItemGroup>
     <PackageReference Include="Microsoft.ApplicationInsights" Version="2.14.0" />
-    <PackageReference Include="Microsoft.Azure.EventHubs" Version="1.0.1" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.7.0" />    
+    <PackageReference Include="Microsoft.Azure.EventHubs" Version="4.3.1" />
```
Comment (karolz-ms) [inline test/Microsoft.Diagnostics.EventFlow.Outputs.Tests/Microsoft.Diagnostics.EventFlow.Outputs.Tests.csproj:30]: The explicit reference to Microsoft.Azure.EventHubs can probably be dropped from this project.

**Review by karolz-ms — APPROVED** [review]
Thanks, Yanming!

File: src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj (hunk 1 above)
Comment (yantang-msft) [reply src/Microsoft.Diagnostics.EventFlow.Outputs.EventHub/Microsoft.Diagnostics.EventFlow.Outputs.EventHub.csproj:7]: That's a good point

**Pull request merged**